etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/database/PostgresDao.java

```diff
@@ -23,55 +21,16 @@ public class PostgresDao implements DbDao {
     private static final PostgresDao INSTANCE = new PostgresDao();
 
     @Inject Logger logger;
```

Comment:
We can delete this line.  We aren't logging anymore in this class.

Reply:
Done